src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -287,6 +271,26 @@ void RendererSettings::apply_settings_to_final_config(asr::Project& project) con
     params.insert_path("passes", m_passes);
     params.insert_path("shading_result_framebuffer", m_passes == 1 ? "ephemeral" : "permanent");
 
+    if (m_lighting_algorithm == 1)
```

Comment:
Something is asymmetric here: we always apply path tracing settings, but we only apply SPPM settings when they are enabled.

I suggest to always apply all settings, and to only set `lighting_engine` to the right value. That way, if the user loads the exported scene in appleseed.studio, he can switch to SPPM and all the settings will be there.

Reply:
True, good idea. Moved the SPPM settings to common settings, so it applies them during export. 
Added an addition to the interactive render export settings, so that path tracing is always used, even if SPPM is set as a lighting engine. Otherwise if one would load the scene into appleseed.studio and start interactive rendering it would crash as it cannot be used with SPPM.